sandy/sections/mf2.py

```diff
@@ -182,272 +186,333 @@ def read_mf2(tape, mat):
                 LRU0.update(header2)
                 C, i = sandy.read_cont(df, i)
                 add = {
-                        "SPI": C.C1,        #Spin, I, of the target nucleus.
-                        "AP": C.C2,         #Scattering radius in units of 10e-12cm.
-                        "NLS": C.N1,        #Number of l-values (neutron orbital angular momentum) in this energy region.
-                        }
+                    "SPI": C.C1,  # Spin, I, of the target nucleus.
+                    "AP": C.C2,  # Scattering radius in units of 10e-12cm.
+                    # Number of l-values (neutron orbital angular momentum) in
+                    # this energy region.
+                    "NLS": C.N1,
+                }
                 LRU0.update(add)
                 dico[(EL, EH)] = LRU0
             elif LRU == 1:
-               if LRF == 1 or LRF == 2 :
-                    if NRO == 0 :
+                if LRF == 1 or LRF == 2:
```

Comment:
Is there an else for this? If not there should be. If it is not accepted by the ENDF-6 format then raise an error